frontend/ui/widget/touchmenu.lua

```diff
@@ -168,11 +170,19 @@ function TouchMenuItem:onTapSelect(arg, ges)
     end
     if enabled == false then return end
 
+    local tap_on_checkmark = false
+    if ges and ges.pos and ges.pos.x then
+        local tap_x = BD.mirroredUILayout() and self.dimen.w - ges.pos.x or ges.pos.x
```

Comment:
nit: `self.dimen.w - ges.pos.x - 1` (dimensions start at 1, positions start at 0)

(I imagine we get it wrong in a host of other places, though ;)).

Reply:
That's not a nit, that's my worst _nitmare_, something that follows me (mostly in crengine) and that I always stumble on and always end up drawing squares and lines on a post-it...
So, I'll trust you blindly on that one :)

Reply:
Hey, I fixed it in the Kobo touch shenanigans, like, a few weeks ago, so, you're not alone ;p.